### ISSUE TITLE
Consider Media Playback (Audio, Video) next?

### ISSUE THREAD

**Issue by rezamohamed** [ewerspej/maui-samples#5]
Could you show a demo of the Media Playback (Audio, Video) next, please?

**Comment by ewerspej** [comment]
Do you mean in the blog or in the repository? Video playback is already used [here](https://github.com/ewerspej/maui-samples/blob/main/MauiSamples/MauiSamples/Views/VideoPage.xaml).

**Comment by rezamohamed** [comment]
My apologies, I should have clarified...more so the recording and playback of audio. I believe you said you were going to write a blogpost about that.

**Comment by ewerspej** [comment]
Hi @rezamohamed,

I'm eventually going to write about playback of video and audio, although I already scratched the surface there with another blog post and some sample code. I haven't taken recording of audio or video into consideration, yet. It does seem like a useful topic, though.

Right now, I have other priorities, so I cannot promise when I will pick it up. I added it to my list of topics. Thank your for the input.

Cheers
Julian
